database/mongodb/mongodb_test.go

```diff
@@ -215,3 +227,41 @@ func TestTransaction(t *testing.T) {
 			}
 		})
 }
+
+type replicaSetStatus struct {
+	Members []replicaMember `bson:"members"`
+}
+
+type replicaMember struct {
+	StateStr string `bson:"stateStr"`
+}
+
+func waitForReplicaInit(client *mongo.Client) error {
+	ticker := time.NewTicker(time.Second * 5)
```

Comment:
Why not use a 1 sec ticker?

Reply:
I did not notice this for 5 seconds ticker, but when I set it for 1 second I saw that the status returns that the node became the primary, but in fact it is a secondary, it takes ~1-2 seconds to rebuild. Just changed check from rs.status to isMaster.